src/components/services-view/services-view.controller.ts

```diff
@@ -3,35 +3,60 @@ import * as _ from 'lodash';
 import * as $ from 'jquery';
 
 export class ServicesViewController implements angular.IController {
-  static $inject = ['Constants', 'Catalog', 'Logger', 'HTMLService', '$filter', '$scope', '$timeout'];
+  static $inject = ['Constants', 'Catalog', 'KeywordService', 'Logger', 'HTMLService', '$filter', '$rootScope', '$scope', '$timeout'];
 
   public ctrl: any = this;
   private constants: any;
   private catalog: any;
+  private keywordService: any;
   private logger: any;
   private htmlService: any;
   private $filter: any;
+  private $rootScope: any;
   private $scope: any;
   private $timeout: any;
   private debounceResize: any;
+  private keyWordFilterFld: any = {
+    id: 'keyword',
+    title:  'Keyword',
+    placeholder: 'Filter by keyword in Category',
```

Comment:
Casing looks odd to me, is this what you meant @beanh66 ?

The placeholder in the search is `Search Catalog` so should we just always use headline case?